sig-community-experience/charter.md

```diff
@@ -0,0 +1,94 @@
+# SIG YOURSIG Charter
+* prefilled
+This charter adheres to the conventions described in the [Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
```

Comment:
I know it's a WIP, but a quick note/reminder to fill that in

Reply:
thanks